install/infra/modules/gke/cluster.tf

```diff
@@ -93,8 +94,8 @@ resource "google_container_node_pool" "services" {
   }
 }
 
-resource "google_container_node_pool" "workspaces" {
-  name               = "workspaces-${var.cluster_name}"
+resource "google_container_node_pool" "regular-workspaces" {
+  name               = "regular-workspaces-${var.cluster_name}"
```

Comment:
Just wondering if this reduces the maximum length allowed for `cluster_name`

Reply:
Hmm, likely  - we can trim this down a bit. Good call!